website/src/components/Layout/Header/index.tsx

```diff
@@ -31,6 +31,9 @@ export const Header: React.FC = () => {
             <ul className="dropdown-menu bg-light">
               <Link to="/projects" className="nav-link navigation-link text-center text-secondary">Projects</Link>
               <Link to="/publications" className="nav-link navigation-link text-center text-secondary">Publications</Link>
+              <li className="mx-5 mx-md-4 nav-item">
+                <Link to="/trap" className="nav-link navigation-link">Trap</Link>
```

Comment:
you forgot some classes here to have the same style as projects and publications